zrml/swaps/src/lib.rs

```diff
@@ -807,6 +809,8 @@ mod pallet {
         UnsupportedTrade,
         /// The outcome asset specified as the winning asset was not found in the pool.
         WinningAssetNotFound,
+        /// Liquidity provided to new Balancer pool is less than `MinLiquidity`.
+        InsufficientLiquidity,
```

Comment:
Let's maintain the lexicographical ordering within the event enum.

Reply:
:thumbsup: